src/components/admin/Select/Select.tsx

```diff
@@ -0,0 +1,54 @@
+import { FieldError, UseFormRegister } from "react-hook-form";
+
+import { AnyForm } from "types/utility";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+export interface Option {
+  name: string;
+  value: string;
+}
+
+type SelectProps = {
+  options: WithId<AnyVenue, string>[];
```

Comment:
Maybe this can be tried. Might show up errors, due to `string` and `SpaceId` not being the same time.

```suggestion
  options: SpaceWithId[];
```

Reply:
[3469e91](https://github.com/sparkletown/sparkle/pull/2944/commits/3469e91693f840fa4d04232bc8c99b452b8f6b6d)